File: java/iniciante/URI1046.java
package URI;

import java.util.Scanner;

public class URI1046 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int start = scanner.nextInt();
        int end = scanner.nextInt();

        if (end > start) {
            int horas = end - start;
            System.out.printf("O JOGO DUROU %d HORA(S\n", horas);
        } else if (end < start) {
            int horas = 24 - (start - end);
            System.out.printf("O JOGO DUROU %d HORA(S)\n", horas);
        } else {
            System.out.println("O JOGO DUROU 24 HORA(S)");
        }
    }
}

File: python/Iniciante/URI1046.py
h_inicial, h_final = map(int, input().split())

if h_inicial > h_final:
  print(f"O JOGO DUROU {24 - (h_inicial - h_final)} HORA(S)")
elif h_final > h_inicial:
  print(f"O JOGO DUROU {h_final - h_inicial} HORA(S)")
else:
  print("O JOGO DUROU 24 HORA(S)")